leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -810,19 +686,28 @@ private int getDefaultZoom() {
     return zoom;
   }
 
-  private void retrieveLocationFrom(String query) {
-    if (searchZone != null && !searchZone.isEmpty()) {
-      retrieveLocationFromZone(query, searchZone);
+  @Override
+  public void retrieveLocationFrom(String query, boolean debounce) {
+    if (debounce) {
+      if (searchZone != null && !searchZone.isEmpty()) {
+        retrieveDebouncedLocationFromZone(query, searchZone, SEARCH_DEBOUNCE_TIME);
```

Comment:
What do you think about moving `SEARCH_DEBOUNCE_TIME` to `retrieveDebouncedLocationFromZone`? It seems that it's not something that it's going to change. So, making it a variable just adds complexity...